BSP/Simulator/Src/BSP_PLL.c

```diff
@@ -33,3 +33,31 @@ uint32_t BSP_PLL_GetSystemClock(void) {
     }
     return SystemCoreClock;
 }
+
+/**
+ * @brief   Delays in param micro seconds
+ * @param   # of microseconds to delay
+ * @return  None
+*/
+void BSP_PLL_delay_u(uint16_t micro)
```

Comment:
I'm not sure we need these delays for the simulator, I think having ```return``` would suffice